### PR TITLE
Brandon/various style updates

### PR DESCRIPTION
**Pull request by brandonhaslegs**

### CODE REVIEW

File: docs/styles/forms.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+section#get-the-app p, section#get-the-app form {
```
Comment (juliendonck) [inline docs/styles/forms.css:1]: This doesn't fit in forms, in my opinion. Should just be in `style.css`

File: docs/styles/typography.css [hunk 2]
```diff
@@ -10,15 +10,15 @@
 
 :root {
   --font-stack: Inter, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+  font-family: var(--font-stack);
```
Comment (juliendonck) [inline docs/styles/typography.css:13]: Should this not be in `body {...}`?

File: pages/index.html.mustache [hunk 3]
```diff
@@ -5,10 +5,15 @@
   {{> header }}
   <div class="content">
     <section id="subtitle">
-      <h1 class="mono">The fastest way to collaborate on code.</h1>
+      <h1 class="xl">The fastest way to collaborate on code.</h1>
     </section>
     <section id="get-the-app">
-      <button>get the app</button>
+      <div id="email signup">
```
Comment (cloudhead) [inline pages/index.html.mustache:11]: I don't know what's going on here, but you shouldn't have a space in your ids.

File: docs/styles/typography.css [hunk 4]
```diff
@@ -10,15 +10,15 @@
...
 }
 
 pre>code {
```
Comment (cloudhead) [inline docs/styles/typography.css:16]:
```suggestion
pre > code {
```

File: docs/styles/forms.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+button {
+    background-color: var(--color-primary);
+    color: var(--color-background);
+    height: 40px;
+    padding: 0 12px;
+    border-radius: 4px;
+    margin-left: 8px;
+}
```
Comment (cloudhead) [inline docs/styles/forms.css:16]: Missing new-line at end of file. Your editor should add this automatically on save :thinking:

File: pages/index.html.mustache (hunk 3 above)
Comment (brandonhaslegs) [reply pages/index.html.mustache:11]: oops that was unintentional

Comment (juliendonck) [inline pages/index.html.mustache:11]:
```suggestion
      <div id="email-signup">
```

**Comment by brandonhaslegs** [comment]
@juliendonck Updated, can you review again

**Pull request merged**